ci/Makefile

```diff
@@ -55,10 +55,6 @@ cleanup:
 gather_logs:
 	${TEST_RUNNER} --platform ${PLATFORM} get_logs
 
-.PHONY: git_rebase_check
-git_rebase_check:
-	${TASKS}/${REBASE}
```

Comment:
Then `REBASE := rebase.py` above can also be deleted.

Reply:
Good catch, I'll do in a follow-up PR just not to have to ask again for aprovals.

Reply:
Good catch. Thanks